.gitignore

```diff
@@ -2,4 +2,5 @@
 /bin/php-cs-fixer
 /bin/phpspec
 /vendor
+.php_cs.cache
```

Comment:
Is it necessary or we can add this into .gitignore?

Reply:
I would use `.*.cache` instead, to catch phpunit cache too (and other possible kinds of cache)